### PR TITLE
Refactor/merge job actions

### PR DESCRIPTION
**Pull request by henrikbossart**
## What does this pull request change?
Merge the job buttons
## Why is this pull request needed?
requested by babak

### CODE REVIEW

File: example/src/plugins/job-ui-single/pages/JobPlugin.tsx [hunk 1]
```diff
@@ -67,33 +100,155 @@ export const JobPlugin = (props: IUIPlugin) => {
...
-          jobEntityDestination={jobEntityDestination}
-          onCreate={(jobEntityId: string) => setJobEntityId(jobEntityId)}
-          defaultJobOutputTarget={props.idReference + '.signal'}
-          defaultJobEntity={defaultJobEntity}
-        />
+    <Card elevation={'raised'} style={{ padding: '1.25rem' }}>
+      <JobButtonWrapper>
+        {jobExists ? (
+          <>
+            <JobControlButton jobStatus={status} start={start} halt={remove} />
```
Comment (ingeridhellen) [inline example/src/plugins/job-ui-single/pages/JobPlugin.tsx:188]: Perhaps clear the results when the button is clicked?

File: packages/dm-core/src/hooks/useJob.tsx [hunk 2]
```diff
@@ -125,6 +125,7 @@ export function useJob(entityId?: string, jobId?: string): IUseJob {
     return dmJobApi
       .startJob({ jobDmssId: entityId })
       .then((response: AxiosResponse<StartJobResponse>) => {
+        console.log(response)
```
Comment (ingeridhellen) [inline packages/dm-core/src/hooks/useJob.tsx:128]:
```suggestion
```

**Review by ingeridhellen — APPROVED** [review]
Looks a lot nicer than before! I find the logic slightly confusing, but I suspect that isn't easily avoidable

**Comment by henrikbossart** [comment]
> I find the logic slightly confusing

Thats because it _is_ confusing..

**Pull request merged**